static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -80,6 +133,10 @@
     }
 }
 
+.content__wrapper--standfirst--immersive-article {
```

Comment:
Is `standfirst` supposed to be a modifier here? Should it be `has-standfirst` just to differentiate it from the `__standfirst` element above?